pytest_motor/mongod_binary.py

```diff
@@ -0,0 +1,132 @@
+"""This module helps with mongodb binary."""
+import platform
+import shutil
+import tarfile
+import tempfile
+import warnings
+from pathlib import Path
+from zipfile import ZipFile
+
+import aiohttp
+import distro
+
+
+class MongodBinary:
+    """This class helps with mongodb binary."""
+    MONGO_VERSION: str = '4.4.6'  # FUTURE: mongod binary version selector via config
+
+    def __init__(self, destination: Path):
+        destination.mkdir(parents=True, exist_ok=True)
+        self.path: Path = destination / f'mongod{".exe" if platform.system() == "Windows" else ""}'
```

Comment:
Maybe refactor `f'mongod{".exe" if platform.system() == "Windows" else ""}'` into a `binary_name` property of `MongodBinary`?